### PR TITLE
Implement updated DrawTools design

### PR DESCRIPTION
**Pull request by shayanypn**
I done with this implementation @aminroosta

### CODE REVIEW

**Comment by aminroosta** [comment]
@brucebinary please review

**Comment by brucebinary** [comment]
LGTM 👍

**Comment by brucebinary** [comment]
wait.

File: src/components/DrawTools.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { connect } from '../store/Connect';
 import Menu from './Menu.jsx';
 import List from './List.jsx';
-import { DrawIcon, DeleteIcon, MeasureIcon } from './Icons.jsx';
+import { DrawIcon, DeleteIcon,ClearIcon, MeasureIcon } from './Icons.jsx';
```
Comment (brucebinary) [inline src/components/DrawTools.jsx]: `DeleteIcon` should be good to remove I think.

**Pull request merged**